### DISCUSSION TITLE
Learn ruby programming language

### DISCUSSION THREAD

**Discussion by Epeoplegit** [community/community#21459] [Other Feature Feedback, Questions, & Ideas]
<p>Hello there .I am a completely begginer for ruby.this is my first programming language of me.which free ebooks is good for me ? Please help me,I read few articles about "how much"useful ruby language.please suggest me some ebooks.<br>
Thank you</p>

**Comment by fire-eggs** [comment]
<p><a href="https://rubybooks.github.io/" rel="noopener nofollow ugc">https://rubybooks.github.io/</a></p>

**Comment by CRO-THEHACKER** [comment, marked as answer]
<p>Check out:</p>
<ul>
<li><a href="https://www.ruby-lang.org/en/documentation/quickstart/" class="inline-onebox" rel="noopener nofollow ugc">Ruby in Twenty Minutes</a></li>
<li>
<a href="https://www.udemy.com/course/ruby-for-absolute-beginners/?LSNPUBID=JVFxdTr9V80&amp;ranEAID=JVFxdTr9V80&amp;ranMID=39197&amp;ranSiteID=JVFxdTr9V80-I4x6sqdY6Cq5ZnOQy75uuw&amp;utm_medium=udemyads&amp;utm_source=aff-campaign" class="inline-onebox" rel="noopener nofollow ugc">Free Ruby Tutorial - Ruby for Absolute Beginners! | Udemy</a> (free udemy course)</li>
</ul>
<p>(if I wasn’t a new user, I would link more, sorry!)</p>
<p>Hope this helps!!</p>
